README.md

```diff
@@ -297,4 +297,4 @@ You should have received a copy of the GNU General Public License
 along with this program. If not, see <https://www.gnu.org/licenses/>.
 ```
 
-Full license [here](https://github.com/CzechPMDevs/MultiWorld/blob/PocketMine-MP/LICENSE).
```

Comment:
This link exists but will give an message: "Branch PocketMine-MP was renamed to master."